cmd/nvidia-cdi-hook/update-nvidia-params/update-nvidia-params.go

```diff
@@ -0,0 +1,199 @@
+/**
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+**/
+
+package nvidiaparams
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/NVIDIA/nvidia-container-toolkit/internal/logger"
+	"github.com/NVIDIA/nvidia-container-toolkit/internal/oci"
+)
+
+const (
+	nvidiaDriverParamsPath = "/proc/driver/nvidia/params"
+)
+
+type command struct {
+	logger logger.Interface
+}
+
+type options struct {
+	containerSpec string
+}
+
+// NewCommand constructs an update-nvidia-params command with the specified logger
+func NewCommand(logger logger.Interface) *cli.Command {
+	c := command{
+		logger: logger,
+	}
+	return c.build()
+}
+
+// build the update-nvidia-params command
+func (m command) build() *cli.Command {
+	cfg := options{}
+
+	// Create the 'update-nvidia-params' command
+	c := cli.Command{
+		Name:  "update-nvidia-params",
```

Comment:
I think we should call this `disable-device-node-modification` or something more descriptive of the intent.

cc @klueska what do you think?

Reply:
Alternatively, I would also be in favor of keeping the `update-nvidia-params` name and adding subcommands or command-line-flags specific to the `update-nvidia-params` command that indicate which parameter in the params file we are updating. This way, if we ever need to update other parameters we have a natural way to express that.

Reply:
I don't want to have multi-modal hooks. We already have a two or three level nesting from the `nvidia-cdi-hook` or `nvidia-ctk hook` commands.